wiki/markdown/cliUsage.md

```diff
@@ -0,0 +1,180 @@
+
+**If you want to have the message publishing logs add d flag:**
+
+```
+java -jar publish-cli.jar -f publishMessages.json -en mb101-eiffel1001 -mb mb101 -domain eiffel001 -mp eiffelsemantics -d
+```
+
+**If you want to have the message publishing on RabbitMQ secured port (5671):**
+-tls option is used to run REMReM on RabbitMQ secured port (5671).
+-tls option value is either 'default' or <version> for secured port.
+
+```
+java -jar publish-cli.jar -f publishMessages.json -en mb101-eiffel1001 -mb mb101 -domain eiffel001 -mp eiffelsemantics -tls 1.2
+```
+
+**NOTE:** for RabbitMQ default port 5672, no need to pass -tls option.
+
+**If you want to have the message publishing on RabbitMQ with given routing key:**
+
+```
+java -jar publish-cli.jar -f publishMessages.json -en mb101-eiffel1001 -mb mb101 -domain eiffel001 -mp eiffelsemantics -rk myroutingkey
+```
+
+**If you want to have the message publishing on RabbitMQ with given tag:**
+
+```
+java -jar publish-cli.jar -f publishMessages.json -en mb101-eiffel1001 -mb mb101 -domain eiffel001 -mp eiffelsemantics -tag production
+```
+
+**To create an exchange passed while publishing the messages use --create_exchange or -ce property to true. Example:**
+
+```
+java -jar publish-cli.jar -f EiffelActivityStartedEvent.json -en mb101-eiffel1001 -ce true -mb mb101 -domain eiffel001 -mp eiffelsemantics
+```
+
+Output for these Example:
+
+```
+[
+  {
+    "id": "963da060-f2cf-4370-a68d-67fd872def36",
+    "status_code": 200,
+    "result": "SUCCESS",
+    "message": "Event sent successfully"
+  }
+]
+```
+
+Output when exchange is not found in RabbitMQ, --create_exchange is set to false , Example:
+
+```
+[
+  {
+    "status_code": 404,
+    "message": " Unavailable. To create the exchange specify -ce or --create_exchange to true
+  }
+]
+```
+
+## Exit Codes
+If CLI fails internally before trying to publish Eiffel message, user will get exit code. Exit codes are described in the table below.
+
+<table>
+    <tbody>
+        <tr>
+          <th style="text-align: left;">Exit code</th>
+          <th style="text-align: left;">Description</th>
+        </tr>
+        <tr>
+          <td>1</td>
+          <td>User will get this exit code in case of some error that is not described below</td>
+        </tr>
+        <tr>
+          <td>2</td>
+          <td>Some CLI options are missed</td>
+        </tr>
+        <tr>
+          <td>3</td>
+          <td>Passed message protocol or message bus configuration are not correct</td>
+        </tr>
+        <tr>
+          <td>4</td>
+          <td>Passed file path is wrong. File is not found</td>
+        </tr>
+        <tr>
+          <td>5</td>
+          <td>Unable to read content from passed file path</td>
+        </tr>
+        <tr>
+          <td>6</td>
+          <td>Unable to read passed JSON string from command line</td>
+        </tr>
+    </tbody>
+</table>
```

Comment:
Any particular reason not to use [Markdown table markup](https://help.github.com/en/github/writing-on-github/organizing-information-with-tables#creating-a-table)? It's easier on the eyes if reading the raw .md file and arguably less tedious to maintain.